lib/api/cli/action.js

```diff
@@ -40,13 +42,16 @@ Action.prototype.onSuccess = function actionOnSuccess (response) {
 };
 
 Action.prototype.onListenCB = function actionOnListenCB (response) {
+  var request;
   clearTimeout(this.timeoutTimer);
+
+  request = new Request(response.data, response.options);
```

Comment:
Don't understand why we create here a `Request` object with an input `response` data.
Probably the `response` variable should be renamed to avoid confusion, but I need some more explanation on how the Cli actions actually work...